medikanren/config.defaults.scm

```diff
@@ -74,5 +74,11 @@
    ;; "decreases_metabolic_processing_of"
    ))
 
+  ;; Enable this flag to prevent translator-web-server.rkt 
+  ;; and open-api/ from making downstream HTTP requests.  Useful
+  ;; for functional CI tests, and for offline development.
+  (trapi-disable-external-requests?  . #f)
```

Comment:
It might be clearer to call this `trapi-enable-external-requests?` and flip all the conditionals, to avoid thinking in terms of double-negatives.  What do you think?